eoscostarica/gGoods
# UI issues at home page

### Fix UI issues at home page

### What does this PR do?

- Resolve #2 
- Solve Home page UI issues

### Steps to test

1. run `make run`
2. go to localhost:3000
3.  see home page

#### CheckList

- [] Follow proper Markdown format
- [] The content is adequate
- [] The content is available in both english and spanish
- [] I Ran a spell check
